media/inf/shortcuts.inf

```diff
@@ -15,7 +15,7 @@ GamesShortcuts=2, %GAMES%
 QuickLaunchShortcuts=26, Microsoft\Internet Explorer\Quick Launch
 
 [DesktopShortcuts]
-%SystemRoot%\readme.txt, %README_TITLE%, %README_DESC%
+%SystemRoot%\notepad.exe, %README_TITLE%, %README_DESC%, 0, %SystemRoot%, readme.txt
```

Comment:
Hmm, I wonder if the shortcut now will have the Notepad icon instead of TXT-file icon :slightly_smiling_face:

Reply:
![image](https://user-images.githubusercontent.com/610685/193432189-1c1b1c53-b2fd-4eb1-a483-c7be35e08a37.png)

Reply:
Yes indeed, it has changed. Here is how it used to look:

![image](https://user-images.githubusercontent.com/578406/193432239-1ed6a41f-2b20-4527-90c2-21a63195aa60.png)

Reply:
That's why, when I added this shortcut back in the day, I made it against readme.txt directly.